src/commands/junit/upload.ts

```diff
@@ -41,6 +41,10 @@ export class UploadJUnitXMLCommand extends Command {
         'Upload all jUnit XML test report files in current directory and add extra tags',
         'datadog-ci junit upload --service my-service --tags key1:value1 --tags key2:value2 .',
       ],
+      [
+        'Upload all jUnit XML test report files in current directory to the datadoghq.eu site',
+        'DATADOG_SITE=datadoghq.eu datadog-ci junit upload --service my-service --tags key1:value1 --tags key2:value2 .',
```

Comment:
```suggestion
        'DATADOG_SITE=datadoghq.eu datadog-ci junit upload --service my-service .',
```
if we want to highlight the datadog site we probably can drop the `tags`